konig-spreadsheet/src/main/java/io/konig/spreadsheet/WorkbookLoader.java

```diff
@@ -1029,11 +1064,267 @@ private void loadSheet(SheetInfo info) throws SpreadsheetException {
 			case SHEET_SECURITY_TAGS:
 				loadSecurityTags(sheet);
 				break;
+			case SHEET_DATA_DICTIONARY_TEMPLATE:
+				loadDataDictionary(sheet);
+				break;
 				
 			}
 
 		}
 
+		private void loadDataDictionary(Sheet sheet) throws SpreadsheetException {
+			readDataDictionaryTemplateHeader(sheet);
+
+			int rowSize = sheet.getLastRowNum() + 1;
+			for (int i = sheet.getFirstRowNum() + 1; i < rowSize; i++) {
+				Row row = sheet.getRow(i);
+				loadDataDictionaryTemplateRow(row);
+			}
+			
+		}
+
+		private void loadDataDictionaryTemplateRow(Row row) throws SpreadsheetException {
+			String sourceSystemName = stringValue(row, sourceSystemCol);
+			String shapeIdLocalName = stringValue(row, sourceObjectNameCol);
+			if(shapeIdLocalName==null )
+				return;
+			URI shapeId = getShapeURL(sourceSystemName,shapeIdLocalName);
+			if (shapeId == null)
+				return;
+			String propertyIdValue = stringValue(row, fieldCol);
+			if (propertyIdValue == null) {
+				logger.warn("Shape Id is defined but Property Id is not defined: {}", shapeId.getLocalName());
+				return;
+			}
+			URI propertyId = expandPropertyId(getPropertyBaseURL()+propertyIdValue);
```

Comment:
I believe there should be a '/' between the propertyBaseURL and the fieldValue.

```
sh:path <{propertyBaseURL}/{field}>
```

Reply:
Added '/' between propertyBaseURL and fieldValue.